src/DXRMapFixups/DeusEx/Classes/DXRFixupM04.uc

```diff
@@ -777,6 +783,11 @@ function AnyEntryMapFixes()
 
         GetConversation('DL_JockParkStart').AddFlagRef('PaulInjured_Played', false); // disable "Your brother's hurt pretty bad" infolink if you've already talked to him
 
+        // for some strange reason, it's not necessary to check if you've talked to Paul for this to work as intended
```

Comment:
The conversation has a check for PaulInjured_Played=False, nothing strange.

Does this logic make sense, if you've been into the hotel, but then go back to the street?  Also what about entrance rando, where maybe you're looking for something in particular.

And as with probably most of these cut infolinks... Does this *actually* provide value (or interest) to the player, or is it just more talking?

Reply:
To be honest, it seems more annoying than beneficial.

Revision adds this infolink if you go into the sewers before the raid, which is also a bit goofy, but it's out of the way, and a truly "what are you doing here" place to be.  I'm not saying that's much better, but at least you really have to go out of your way to be bugged.